### PR TITLE
update libocr, solana, terra

### PR DESCRIPTION
**Pull request by aalu1418**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -83,10 +83,15 @@ require (
...
 	gonum.org/v1/gonum v0.11.0
-	google.golang.org/protobuf v1.27.1
+	google.golang.org/protobuf v1.28.0
 	gopkg.in/guregu/null.v4 v4.0.0
 )
 
+require (
+	github.com/codegangsta/negroni v1.0.0 // indirect
+	gopkg.in/guregu/null.v2 v2.1.2 // indirect
+)
```
Comment (jmank88) [inline go.mod:93]: Let's combine these indirect require blocks in to one. I don't know why go mod does this :thinking:

Comment (jmank88) [inline go.mod:93]: Let's combine these indirect require blocks in to one. I don't know why go mod does this :thinking:

**Review by jmank88 — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -312,6 +313,7 @@ require (
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac // indirect
 	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368 // indirect
 	google.golang.org/grpc v1.43.0 // indirect
+	gopkg.in/guregu/null.v2 v2.1.2 // indirect
```
Comment (jmank88) [inline go.mod:316]: Oh wait, we have v4 already - should we be aligned on that?

Comment (aalu1418) [reply go.mod:316]: yeah that would be good...trying to figure out where v2 started showing up

Comment (jmank88) [reply go.mod:316]: actually, not from solana/terra/libocr, and can't figure out where....
```
$ go mod why gopkg.in/guregu/null.v2
# gopkg.in/guregu/null.v2
(main module does not need package gopkg.in/guregu/null.v2)
```
:shrug: https://github.com/golang/go/issues/27900

Comment (jmank88) [reply go.mod:316]: Could be indirect and _test only_? https://github.com/golang/go/issues/30206

Comment (aalu1418) [reply go.mod:316]: ...yeah i'm struggling to find this too. will give it another shot tomorrow 🙃

**Review by jmank88 — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (aalu1418) [reply go.mod:316]: notes: 
* this likely happened because of bumping the core version in solana & terra repos to core 1.13 so go mod chose differently

**Pull request merged**